.ci/gcb-generate-diffs.yml

```diff
@@ -1,8 +1,44 @@
 ---
 steps:
+    - name: 'alpine'
+      args:
```

Comment:
Can we get a short comment on what this does? why do we need sh -c instead of just doing rm?

Reply:
Commented more elaborately.